packages/cactus-plugin-ledger-connector-fabric/README.md

```diff
@@ -223,24 +267,33 @@ Currently Cactus Fabric Connector supports following Identity Providers
 - Vault-X.509 : Secure provider wherein `private` key is stored with vault's transit transit engine and certificate in `certDatastore`. Rather then bringing the key to the connector, message digest are sent to the vault server which returns the `signature`.
 - WS-X.509 : Secure provider wherein `private` key is stored with `client` and certificate in `certDatastore`. To get the fabric messages signed, message digest is sent to the client via `webSocket` connection opened by the client in the beginning (as described above)
 
-### setting up a WS-X.509 provider
+#### 1.4.2. Setting up a WS-X.509 provider
 The following packages are used to access private keys (via web-socket)  stored on a clients external device (e.g., browser, mobile app, or an IoT device...).
   -[ws-identity](https://github.com/brioux/ws-identity): web-socket server that issues new ws-session tickets, authenticates incoming connections, and sends signature requests
   -[ws-identity-client](https://github.com/brioux/ws-identity-client): backend connector to send requests from fabric application to ws-identity
   -[ws-wallet](https://github.com/brioux/ws-wallet): external clients crypto key tool: create new key pair, request session ticket and open web-socket connection with ws-identity
 
-### Building the ws-identity docker image
+#### 1.4.3. Building the ws-identity docker image
+
+TBD
+
+## 2. Architecture
+The sequence diagrams for various endpoints are mentioned below
+
+### 2.1. run-transaction-endpoint
+![run-transaction-endpoint sequence diagram](docs/architecture/images/run-transaction-endpoint.png)
```

Comment:
There should be a new line after this(just like at line 290), as the image and the following text is rendered together
![image](https://user-images.githubusercontent.com/56861721/145160321-d6b2d41d-fc9b-4449-8203-d3a5a8fce35d.png)

Reply:
@jagpreetsinghsasan Oops, thank you, fixing now.